packages/cli/src/util/errors.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @flow
+ */
+const chalk = require('chalk');
+
+class ProcessError extends Error {
+  constructor(msg: string, processError: string) {
+    super(`${chalk.red(msg)}\n\n${chalk.gray(processError)}`);
```

Comment:
Looks silly, but I'm cool with that :)

Reply:
I would say it's a "temporary" solution when we figure out proper logging. I think it's a good starting point.